src/GogeToken.sol

```diff
@@ -342,7 +342,7 @@ contract DividendPayingToken is ERC20, Ownable, IDividendPayingToken, IDividendP
     //   where `dividendCorrectionOf(_user)` is updated whenever `balanceOf(_user)` is changed:
     //   `dividendCorrectionOf(_user) = dividendPerShare * (old balanceOf(_user)) - (new balanceOf(_user))`.
     // So now `dividendOf(_user)` returns the same value before and after `balanceOf(_user)` is changed.
-    mapping(address => int256) internal magnifiedDividendCorrections;
+    mapping(address => int256) public magnifiedDividendCorrections;
```

Comment:
Why does the `abstract contract IERC20Extended` on line 8 exist? This declaration can be removed and the function signature moved to the `IERC20.sol` interface in `Interfaces.sol` since this contract uses `decimals()`.

Reply:
I actually have no idea. This is the first time seeing this